python/ray/_private/worker.py

```diff
@@ -2171,7 +2196,8 @@ def listen_error_messages(worker, threads_stopped):
                     "lines": [error_message],
                     "pid": "raylet",
                     "is_err": False,
-                }
+                },
+                ignore_prefix,
```

Comment:
This is not worker process log but raylet logs. Let's keep prefix for it now since otherwise we don't know which raylet this log is about.

Reply:
updated https://github.com/ray-project/ray/pull/47703/commits/0556458613658c85a7619ff90e9a8f450c143ebc